electrum/wizard.py

```diff
@@ -704,14 +704,17 @@ def do_configure_server(self, wizard_data: dict):
         self._logger.debug(f'configuring server: {wizard_data!r}')
         net_params = self._daemon.network.get_parameters()
         server = ''
+        oneserver = wizard_data['one_server']
```

Comment:
you also need to set this parameter in the desktop qt gui if you refer to it here.
```
  6.15 | D | wizard | view "server_config" last: True
  6.15 | D | wizard | view=server_config
  6.15 | D | wizard | configuring server: {'use_defaults': False, 'want_proxy': False, 'autoconnect': True, 'server': ''}
  6.15 | E | gui.qt.exception_window.Exception_Hook | exception caught by crash reporter
Traceback (most recent call last):
  File "/home/user/wspace/electrum/electrum/gui/qt/wizard/wizard.py", line 214, in on_next_button_clicked
    self.submit(wd)
  File "/home/user/wspace/electrum/electrum/gui/qt/wizard/wizard.py", line 232, in submit
    view = self.resolve_next(self._current.view, wdata)
  File "/home/user/wspace/electrum/electrum/wizard.py", line 74, in resolve_next
    view_accept(wizard_data)
  File "/home/user/wspace/electrum/electrum/wizard.py", line 707, in do_configure_server
    oneserver = wizard_data['one_server']
KeyError: 'one_server'
```